src/flow_information.rs

```diff
@@ -26,13 +28,15 @@ mod tests {
     #[test]
     fn test_default() {
         let default = FlowInformation::default();
+        assert!(default.sni.is_empty());
```

Comment:
If Option<&str>, let's check the field is there empty/ not empty/not there if possible

Reply:
ok